deploy/cephcsi/image/Dockerfile

```diff
@@ -8,14 +8,14 @@ LABEL stage="build"
 
 ARG CSI_IMAGE_NAME=quay.io/cephcsi/cephcsi
 ARG CSI_IMAGE_VERSION=canary
-ARG GO_ARCH
 ARG SRC_DIR
 ARG GIT_COMMIT
 ARG GOROOT=/usr/local/go
 
 COPY build.env /
 
 RUN source /build.env && \
+    ( test -n "${GO_ARCH}" && exit 0; echo -e "\n\nMissing GO_ARCH argument for building image\n\n"; exit 1 ) && \
```

Comment:
can you update the error message on how to run make image-cephcsi with GO_ARCH flag?

Reply:
sure